src/ripple/app/tx/impl/details/NFTokenUtils.cpp

```diff
@@ -428,10 +428,48 @@ removeToken(
         return tesSUCCESS;
     }
 
-    // The page is empty, so we can just unlink it and then remove it.
     if (prev)
     {
-        // Make our previous page point to our next page:
+        // With fixNFTokenPageLinks...
+        // The page is empty and there is a prev.  If the last page of the
```

Comment:
This comment is double spaced (I noticed the others were too). I don't think double spacing is very common, but I don't object too strongly if you prefer it.